Dockerfile

```diff
@@ -45,20 +46,29 @@ RUN ollama serve & while ! curl http://localhost:11434; do sleep 1; done; ollama
 # Setup the custom API and frontend
 WORKDIR /workspace
 COPY --chmod=644 gswikichat gswikichat
-COPY --chmod=755 frontend frontend
+COPY --chmod=755 static static
 COPY --chmod=755 json_input json_input
 
+
 # Install node from upstream, ubuntu packages are too old
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+
 RUN curl -sL https://deb.nodesource.com/setup_20.x | bash
 RUN apt-get install -y nodejs && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+RUN npm cache clean -f
+RUN npm install -g n
+RUN n 18.17.1
```

Comment:
I assume that this branch PR is no longer relevant because the `integration` branch is more direct and streamlined than this solution

Reply:
Agree.